realtime/realtime.py

```diff
@@ -53,16 +53,21 @@ def get_temp_file_from_datawrite():
 
                 # Make sure we only process the first slice for simulatenous multislice data for now
                 if file_time == last_file_time:
+                    os.remove(filename)
                     continue
 
                 last_file_time = file_time
 
                 slice_num = int(fields[5])
                 try:
+                    rt_print("Using pyDARN to convert {}".format(filename))
                     converted = pydarn.BorealisConvert(filename, "rawacf", "/dev/null", slice_num,
                                                     "site")
-                except:
+                    os.remove(filename)
+                except pydarn.exceptions.borealis_exceptions.BorealisConvert2RawacfError as e:
                     rt_print("Error converting {}".format(filename))
+                    rt_print(str(e))
+                    os.remove(filename)
```

Comment:
os.remove will only raise OSError if filename is a directory, so this should be safe